misp.py

```diff
@@ -961,9 +1015,30 @@ def to_ids(self, value):
         self._to_ids = int(value)
 
     @property
+    def server(self):
+        return self._server
+
+    @to_ids.setter
+    def server(self, value):
+        self._server = value
+
+    @property
     def ShadowAttribute(self):
         return None
 
+    def download(self):
```

Comment:
Open question: Why not move the `download()` method to the `MispServer` class?

```
event = server.events.get(42)
for attr in event.attributes:
      if attr.type == 'malware-sample':
        data = server.download(attr)
```

That way, no need to dance with `_server` attribute in `MispEvent`. What do you think?